Dockerfile

```diff
@@ -0,0 +1,87 @@
+#
+# Dockerfile for an image with the currently checked out version of zipline installed. To build:
+#
+#    docker build -t quantopian/zipline .
+# 
+# To run the container:
+#
+#    docker run -v=/path/to/your/notebooks:/projects -p 8888:8888/tcp --name zipline -it quantopian/zipline
```

Comment:
One thing I noticed while testing this is that if you don't mount an external directory as a volume, you get an error if you try to create a new notebook.  Is that intentional and/or should we warn people that failure creating notebooks likely means failure to mount a local directory?